### PR TITLE
Maximum player option

### PR DESCRIPTION
**Pull request by eskolnik**
This is an attempt to address the 6-player issue raised in #2053 
This PR allows the creator of a table to set a player limit, defaulting to 5. The current / max player count is displayed on each unstarted table in the lobby.

I opted to put the player count option in the main game creation modal, but if we want to add a little friction (and thereby reduce the number of 6p lobbies even further), we could nest it under "Extra Options." 

hanabUI_max_players_create
hanabUI_max_players_lobby
hanabUI_max_players_warning

### CODE REVIEW

File: client/src/lobby/createGame.ts
Comment (timotree3) [inline client/src/lobby/createGame.ts:260]:
```suggestion
  // Max players must be a number between 2 and 6. Default to 5 if validation fails
```

**Review by timotree3 — APPROVED** [review]
Thanks! The code looks great. We'll see if Zamiel wants to merge the feature

**Comment by Zamiell** [comment]
seems fine

**Comment by Zamiell** [comment]
i can't merge any server changes until the channels branch is merged though

**Comment by eskolnik** [comment]
fair enough, I figure just having this PR to point to should be sufficient to (hopefully) resolve the 6p discussion, whether it gets merged now or weeks from now.

**Comment by Zamiell** [comment]
also a semi colon is more grammatically correct than a comma

**Comment by Zamiell** [comment]
looks like this is fixed in #2314

**Pull request closed without merging**
